frontend/main.js

```diff
@@ -1,13 +1,74 @@
 const { app, BrowserWindow, dialog, ipcMain } = require("electron/main");
 const path = require("node:path");
-
+const fs = require("fs-extra");
+const { PythonShell } = require("python-shell");
+const constants = require("./constants/constants.js");
 let mainWindow;
 
+function runProgram(filePath) {
+  mainWindow.loadFile("loading.html");
+
+  const script = constants.SCRIPT;
+
+  const poetryArgs = [
+    constants.COMMANDS.DEDUPE_DATA,
+    constants.OPTIONS.FORMAT,
+    constants.FORMAT.TEST,
+    filePath,
+    "./",
+  ];
```

Comment:
From this it looks like the input file is `filePath` and the output dir is `"./"`. This is good although if we are assuming that the user is passing in a CSV we should change the format to CSV and not TEST.